src/plugins/console/public/lib/autocomplete/components/index_template_autocomplete_component.js

```diff
@@ -6,12 +6,16 @@
  * Side Public License, v 1.
  */
 
-import { getIndexTemplates } from '../../mappings/mappings';
+import { getAutocompleteInfo } from '../../../services';
 import { ListComponent } from './list_component';
 
+function ListGenerator() {
```

Comment:
Honestly not sure that i like it. In fact this method is not a constructor and returns only some data.

I prefer something like 

`super(name, getAutocompleteInfo().getFactoryFor(ACTOCOMPLETE_INFO.indexTemplate), parent, true, true);`

Reply:
or simpler